kubernetes/monitoring/scripts/victoria-metrics-restore.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+if [ $# -lt 2 ]; then
+     echo "Usage Instructions: <bucket-path> <backup service-name>"
+     echo "victoria-metrics-restore.sh cms-monitoring/vmbackup/2021/05/31/00:00 victoria-metrics"
+     echo "victoria-metrics-restore.sh cms-monitoring/vmbackup-long/2021/05/31/00:00 victoria-metrics-long"
+     exit 1;
+fi
+
+bucket_path=$1
+backup_service=$2
+
+# use tmp area to store service file
+tmpDir=/tmp/$USER/k8s/
+if [ -d $tmpDir ]; then
+    rm -rf $tmpDir
+fi
+mkdir -p $tmpDir
+cd $tmpDir
+
+curl -ksLO https://raw.githubusercontent.com/dmwm/CMSKubernetes/master/kubernetes/monitoring/services/agg/vmrestore.yaml
+#ls $tmpDir
+if [[ "$backup_service"  == "victoria-metrics" ]] ; then
+
+	echo "testing"
+  cat vmrestore.yaml | sed -e "s,cms-monitoring/vmbackup/2021/05/31/00:00,$bucket_path,g" > vmrestore.yaml.new
+  
+  curl -ksLO https://raw.githubusercontent.com/dmwm/CMSKubernetes/master/kubernetes/monitoring/services/agg/victoria-metrics.yaml
+
+  kubectl delete -f victoria-metrics.yaml
+  sleep 60s
+
+  kubectl apply -f vmrestore.yaml.new
+
+  sleep 60s
```

Comment:
replace sleep with loop to check conditions of the service. We should use API to check if service is started, or even better kubectl logs to read certain pattern. Here again, make finite loop with some interval, check the conditions and exit appropriately.

For instance, here is what I see from VM startup:
```
2021-05-31T16:45:34.314Z        info    VictoriaMetrics/app/victoria-metrics/main.go:62 started VictoriaMetrics in 0.894 seconds
```
So, the pattern to check would be `.*info.*started VictoriaMetrics.*`